src/Dropdown/Dropdown.js

```diff
@@ -114,7 +115,7 @@ export default function Dropdown({
       <Manager>
         <Reference>
           {({ ref: triggerRef }) => (
-            <TriggerWrapper ref={triggerRef} tabIndex={-1}>
+            <TriggerWrapper style={styles.TriggerWrapper} ref={triggerRef} tabIndex={-1}>
```

Comment:
Nice - can we call this `styles.Trigger` and change component to `<Trigger />` to reduce verbosity

Reply:
Good idea.